src/components/new-request/submit-request/reserve-submit.vue

```diff
@@ -71,22 +71,18 @@ export default class ReserveSubmitButton extends Vue {
     this.cancelAnalyzeName('NamesCapture')
   }
 
-  getData (type: string): any {
-    if (this.getAssumedName) type = 'assumed'
-    let data: any
+  getData (type: NrType): DraftReqI | ConditionalReqI | ReservedReqI {
```

Comment:
I cleaned this up when I intended to use this method to get "RESUBMIT" data, but then I found a better way.